### PR TITLE
support issue international

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind bug

#### What this PR does / why we need it:
support issue international

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/bug?id=206472&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDQzNiJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=431&type=BUG)
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/bug?id=206547&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDQzNiJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=431&type=BUG)

#### Specified Reviewers:

/assign @Effet @ZLValien 

#### Need cherry-pick to release versions?

cherry-pick release/1.2

### CODE REVIEW

**Comment by ZLValien** [comment]
Please specify in the title which pages related to the issue have been changed

File: modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go [hunk 1]
```diff
@@ -43,113 +43,113 @@ type FrontendConditions struct {
...
 }
 
-func generateFrontendConditionProps(fixedIssueType string, state State) FrontendConditionProps {
+func generateFrontendConditionProps(fixedIssueType string, state State, bdl protocol.ContextBundle) FrontendConditionProps {
 	conditionProps := []filter.PropCondition{
 		{
 			Key:         PropConditionKeyIterationIDs,
-			Label:       "迭代",
-			EmptyText:   "全部",
+			Label:       bdl.I18nPrinter.Sprintf("Sprint"),
```
Comment (ZLValien) [inline modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go:50]: iteration?sprint?

File: modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go [hunk 2]
```diff
@@ -217,7 +217,7 @@ func generateFrontendConditionProps(fixedIssueType string, state State) Frontend
 	if fixedIssueType == apistructs.IssueTypeBug.String() {
 		conditionProps = append(conditionProps, filter.PropCondition{
 			Key:         PropConditionKeyClosed,
-			Label:       "关闭日期",
+			Label:       bdl.I18nPrinter.Sprintf("Closed at"),
```
Comment (ZLValien) [inline modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go:220]: case is different with Created At

File: modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go (hunk 1 above)
Comment (QvodSoldier) [reply modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go:50]: The front-end internationalization that does not use componentization uses sprint

File: modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go (hunk 2 above)
Comment (QvodSoldier) [reply modules/openapi/component-protocol/scenarios/issue-manage/components/issueFilter/model.go]: done

File: modules/openapi/component-protocol/scenarios/issue-manage/components/issueGantt/model.go [hunk 3]
```diff
@@ -98,13 +98,13 @@ func (g *Gantt) genProps(edgeMinTime, edgeMaxTime *time.Time) {
 		RowKey:  "id",
 		// ClassName: "task-gantt-table",
 		Columns: []gantt.PropColumn{
-			{Title: "处理人", DataIndex: "user", Width: 160},
-			{Title: "标题", DataIndex: "issues",
+			{Title: g.CtxBdl.I18nPrinter.Sprintf("Gantt Chart"), DataIndex: "user", Width: 160},
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/issue-manage/components/issueGantt/model.go:101]: "Gantt Chart" to replace "处理人"？

File: modules/openapi/component-protocol/scenarios/issue-manage/components/issueTable/render.go [hunk 4]
```diff
@@ -775,3 +775,41 @@ func getTotalPage(total, pageSize uint64) (page uint64) {
...
+        {
+            "width": 110,
+            "dataIndex": "state",
+            "title": ` + "State" + `
+        },
+        {
+            "width": 120,
+            "dataIndex": "assignee",
+            "title": ` + "Assignee" + `
+        },
```
Comment (Effet) [inline modules/openapi/component-protocol/scenarios/issue-manage/components/issueTable/render.go:805]: why you concat string? why not use one string as a json?

**Review by Effet — CHANGES_REQUESTED** [review]

**Review by Effet — APPROVED** [review]

**Comment by QvodSoldier** [comment]
components-protocol refactor

**Pull request closed without merging**